chef_dsl_metadata/README.md

```diff
@@ -1,8 +1,8 @@
 # Generating metadata for a specific chef version
 - run 'bundle install' to install appraisal
 - update Appraisals file with new chef version(s)
-- run 'appraisal chef_12.7.2 bundle install' to generate the Gemfile and Gemfile.lock
-- run 'appraisal chef_12.7.2 rake generate_chef_metadata' to generate json for that version of chef
+- run 'bundle exec appraisal install'. It will fail, but it will still generate the Gemfile
```

Comment:
missing the version on this line or does it no longer work?

not sure why we need to bother bumping to appraisal 2.x either does it actually buy us anything?